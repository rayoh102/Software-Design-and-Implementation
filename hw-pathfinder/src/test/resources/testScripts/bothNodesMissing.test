# Find a path with Dijkstra's algorithm when
# both nodes don't exist

CreateGraph simpleGraph

FindPath simpleGraph A B
