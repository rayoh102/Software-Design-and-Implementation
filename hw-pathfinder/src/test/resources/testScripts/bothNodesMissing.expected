# Find a path with Dijkstra's algorithm when
# both nodes don't exist

created graph simpleGraph

unknown node A
unknown node B